back/src/app.ts

```diff
@@ -35,10 +36,14 @@ const options: cors.CorsOptions = {
 };
```

Comment:
Check core/servers configuration in planning poker about express and cors config: https://github.com/Lemoncode/planning-poker-example/tree/master/back/src/core/servers

Reply:
Nice, I just create #114 issue for that